src/backend/genes.js

```diff
@@ -18,11 +18,15 @@ export const urlGeneList = ({ organism, limit = maxLimit }) => {
   return url;
 };
 
-export const urlGeneSearch = ({ query, limit = defaultLimit }) => {
+export const urlGeneSearch = ({
+  query,
+  exact = false,
+  limit = defaultLimit
+}) => {
   const params = new URLSearchParams();
   params.set('limit', limit);
   if (query)
-    params.set('autocomplete', query);
+    params.set(exact ? 'search' : 'autocomplete', query);
```

Comment:
`search` is actually full text search, not exact search. I will add a new parameter (something like `search-list`, which accepts a comma-separated list) to address it.

Reply:
i dont know the backend terms for this stuff. i dont find "search" or "autocomplete" to be very elucidating either. i just named this "exact" because in my testing it only ever returns a single result.

Reply:
I can call it "precise" instead? I just need to make it clear to my future self, and I guess preferably without colliding with established backend terminology.

Also, are you talking about adding a single endpoint for multiple queries at the same time? When I was first making the app I would've asked for that, but now that I've implemented individual, multiple concurrent queries, I kinda like it that way. That way, the user gets a loading spinner for each query (instead of one big one for all of the queries), and sort of gets a progress update.

Reply:
like this

![Q0wEjQCZ95](https://user-images.githubusercontent.com/8326331/81199428-a2f13200-8f90-11ea-9144-18e1102ee026.gif)

Reply:
Either `exact` or `precise` is good to me. (I prefer `exact` a little more.) 

The spinner for each search query seems fancy, but when a user types in a list of genes to search, I am not sure how much he cares about the individual search query's result. If you like it, I can implement an API parameter like `exact-match=xxx` which matches only genes whose names or aliases match `xxx` exactly (case-insensitive), instead of treating `xxx` as a comma-separated list. I will let you know when it is done.

Reply:
When you have a really long list like in #177, it takes a while to query them all, and individual spinners to show progress is good. Unless combining the searches on the backend would be significantly faster.

So if I understand correctly, "exact" search means it looks for fields that essentially equal the query string (case insensitive), with nothing more and nothing less. And "full text" is more like substring matching, where you just find entries who contain that string in them. And then "autocomplete" is some sort of fuzzy search.

Idk if we need an exact search. I think substring/full-text might be enough. Let's what Deb thinks when testing out this PR.

Reply:
Full text search is exact word search with word stems considered:
https://www.postgresql.org/docs/12/textsearch.html

I will hold the API change and wait for more user comments.